dev_guide/templates.adoc

```diff
@@ -10,7 +10,7 @@
 toc::[]
 
 == Overview
-A template describes a set of resources intended to be used together that can be customized and processed to produce a configuration. Each template defines a list of parameters that can be modified for consumption by containers. It also defines a set of labels to apply to every resource created by the template.
+A template describes a set of resources intended to be used together that can be customized and processed to produce a configuration. Each template defines a set of parameters that can be modified for consumption by containers, and defines a set of labels to apply to every resource created by the template.
```

Comment:
It would be nice to be linked here (probably on the word "resources" in the first sentence) to something that helps inform or remind what those resources could be. Maybe the best target right now would be http://docs.openshift.org/latest/architecture/core_objects/overview.html. It's a stub at the moment, but it would at least get them in the right subdir (to then check out `kubernetes_model.adoc` and `openshift_model.adoc`), and we can add some info there to give context.

Also a reminder to try and hard-wrap this line at 80 while you're in the file.